product_docs/docs/pem/9/considerations/pem_pgbouncer/preparing_the_pem_database_server.mdx

```diff
@@ -65,14 +82,65 @@ You must configure the PEM database server to work with PgBouncer. This example
     (1 row)
     ```
 
-    The function creates a user with the same name and a random password and grants pem_agent and pem_agent_pool roles to the user. This approach allows pgBouncer to use a proxy user on behalf of the agent.
+    The function creates a user with the same name and a random password and grants pem_agent and pem_agent_pool roles to the user. This approach allows PgBouncer to use a proxy user on behalf of the agent.
+
+## Updating the configuration files to allow PgBouncer-PEM connections
+
+1.  Allow the pgbouncer user to connect to the `pem` database using the SSL authentication method by adding the following entry to the `pg_hba.conf` file of the PEM database server:
 
-7.  Add the following entries to the start of the `pg_hba.conf` file of the PEM database server. These entries allow the pgBouncer user to connect to the `pem` database using the md5 authentication method.
+    !!!note
+       Add the following entry before any `+pem_agent` entries
```

Comment:
This note is confusing. It doesn't have any ending punctuation and it is not clear when "the following entry" refers to here.